source/server/hot_restart_impl.h

```diff
@@ -120,8 +120,9 @@ class HotRestartImpl : public HotRestart,
 public:
   HotRestartImpl(Options& options);
 
-  Thread::BasicLockable& logLock() { return log_lock_; }
-  Thread::BasicLockable& accessLogLock() { return access_log_lock_; }
+  Thread::BasicLockable& logLock() override { return log_lock_; }
```

Comment:
nit: Can you move these into the block under 127 below. (This is the same nit I commented on before in the other file, which I didn't read carefully enough, sorry). The idea is just to visually group the overrides so it's really easy to see which are overrides for which base class and which are free functions.

Reply:
done